src/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.block-1 {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.block-2 {
+  background-color: rgb(80%, 0%, 0%);
+  order: 1;
+}
+
+.block-3 {
+  background-color: rgb(60%, 0%, 0%);
+  order: 1;
+}
+
+.block-4 {
+  background-color: rgb(40%, 0%, 0%);
+  order: 1;
+}
+
+.block-5 {
+  background-color: rgb(20%, 0%, 0%);
+  order: 1;
+}
+
+.block-6 {
+  background-color: rgb(0%, 0%, 0%);
+  order: 1;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.block {
+  height: 300px;
+  /* width: 300px; */
+  font-size: 100px;
+  color: white;
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 300px;
+}
+
+@media (min-width: 599px) {
```

Comment:
that's what i see on `599` breakpoint
![image](https://user-images.githubusercontent.com/37161315/167645876-a7de9267-3608-4e17-a3d9-bc2bc4c7f190.png)